webui/react/src/pages/TrialDetails/LogViewer.tsx

```diff
@@ -0,0 +1,527 @@
+            },
+            { even: 0, odd: 0 },
+          );
+        const isEven = oddOrEven.even > oddOrEven.odd;
+        const content = lines.reduce((acc, line, index) => {
+          const skipNewline = (isEven && index % 2 === 0) || (!isEven && index % 2 === 1);
+          return acc + line + (skipNewline ? ' ' : '\n');
+        }, '');
+        e.clipboardData?.setData(clipboardFormat, content);
+      }
+      e.preventDefault();
+    };
+
+    target.addEventListener('copy', handleCopy);
+
+    return (): void => target?.removeEventListener('copy', handleCopy);
+  }, [logsRef]);
+
+  return (
+    <Section fullHeight>
+      <div className={css.options}>
+        <Row>
+          <Column width="shrink">{title}</Column>
+          <Column align="right">
+            <Row>
+              <ClipboardButton
+                copiedMessage={clipboardCopiedMessage}
+                getContent={getClipboardContent}
+              />
+              <Button
+                aria-label="Toggle Fullscreen Mode"
+                icon={<Icon name="fullscreen" showTooltip title="Toggle Fullscreen Mode" />}
+                onClick={handleFullScreen}
+              />
+              {handleCloseLogs && (
+                <Button
+                  aria-label="Close Logs"
+                  icon={<Icon name="close" title="Close Logs" />}
+                  onClick={handleCloseLogs}
+                />
+              )}
+              {onDownload && (
+                <Button
+                  aria-label="Download Logs"
+                  icon={<Icon name="download" showTooltip title="Download Logs" />}
+                  onClick={handleDownload}
+                />
+              )}
+            </Row>
+          </Column>
+        </Row>
+      </div>
+      <Spinner center spinning={isFetching}>
+        <div className={css.base} style={{ height: height === 'stretch' ? '100%' : `${height}px` }}>
+          <div className={css.logs} ref={logsRef}>
+            {logs.length > 0 ? (
+              <Virtuoso
+                atBottomStateChange={handleReachedBottom}
+                atBottomThreshold={12}
+                atTopStateChange={handleReachedTop}
+                customScrollParent={logsRef.current || undefined}
+                data={logs}
+                firstItemIndex={firstItemIndex}
+                followOutput="smooth"
+                initialTopMostItemIndex={
+                  fetchDirection === FetchDirection.Older
+                    ? (initialLogs?.length ?? PAGE_LIMIT) - 1
+                    : 0
+                }
+                itemContent={(_index, logEntry) => (
+                  <LogViewerEntry
+                    formattedTime={logEntry.formattedTime}
+                    key={logEntry.id}
+                    level={logEntry.level}
+                    message={logEntry.message}
+                    style={{
+                      backgroundColor: logEntry.id === selectedLog?.id ? '#E7F7FF' : 'transparent',
```

Comment:
see above comment re: putting this color in css instead. it could be a css variable.

Reply:
<img width="1443" alt="Screenshot 2024-10-07 at 11 50 54 AM" src="https://github.com/user-attachments/assets/f305e6c5-5a16-416d-b6a9-28a8607bc4c0">
I tried to use the color variable, it looks normal on light mode, but on dark mode, the color does not change..

Reply:
seems like it is changing, just really dark. you can also see that it has the same background color as the selected log entry in the search age. It might make sense to use the `status-active`/`status-active-on` variables?

Reply:
For log level bug, we already have the font color to be blue, so I doubt it would make enough contrast. 
What I'm curious is that I used the same color variable on both sides, and I see a really dark highlight at the left panel, but the right panel still shows the light highlight...